azure_functions_worker/dispatcher.py

```diff
@@ -163,7 +170,7 @@ def on_logging(self, record: logging.LogRecord, formatted_msg: str) -> None:
 
         if is_system_log_category(record.name):
             log_category = protos.RpcLog.RpcLogCategory.System
-        else:
+        else:  # import logging; logging.info will yield 'root' in record.name
```

Comment:
Can you clarify why is this comment added?

Reply:
A lot of users using our basic template
```python
import logging
logging.info('some message')
```
Will use the **root** logger for logging, so we mark `record.name == 'root'` as protos.RpcLog.RpcLogCategory.User. This comment is here to explain why root logger belongs to User logs.